Services/CO.CDP.OrganisationInformation.Persistence/OrganisationInformationContext.cs

```diff
@@ -16,9 +17,38 @@ public class OrganisationInformationContext(DbContextOptions<OrganisationInforma
     public DbSet<Person> Persons { get; set; } = null!;
     public DbSet<Form> Forms { get; set; } = null!;
     public DbSet<SharedConsent> SharedConsents { get; set; } = null!;
+    public DbSet<ConnectedEntity> ConnectedEntities { get; set; } = null!;
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
+        modelBuilder.HasPostgresEnum<ControlCondition>();
+        modelBuilder.HasPostgresEnum<ConnectedEntityType>();
+        modelBuilder.HasPostgresEnum<ConnectedPersonType>();
+        modelBuilder.HasPostgresEnum<ConnectedPersonCategory>();
+        modelBuilder.HasPostgresEnum<ConnectedOrganisationCategory>();
```

Comment:
This isn't enough to take full advantage of enums in postgres, but it's a good start 👍